### ISSUE TITLE
Scala 2.12 RC2

### ISSUE THREAD

**Issue by schmitch** [etorreborre/specs2#517]
Once again, sorry to bother you.

Would be great to have a release against Scala 2.12 RC2.

**Comment by etorreborre** [comment]
I have planned to do release a new version this week but I'm waiting for `scalaz` to publish their own version first.

**Comment by xuwei-k** [comment]
I just published scalaz 7.2.6 and 7.1.10 for Scala 2.12.0-RC2 (without scalacheck-binding).
- https://oss.sonatype.org/content/repositories/releases/org/scalaz/scalaz-core_2.12.0-RC2/7.2.6/
- https://oss.sonatype.org/content/repositories/releases/org/scalaz/scalaz-core_2.12.0-RC2/7.1.10/

**Comment by etorreborre** [comment]
Thanks!

**Comment by larsrh** [comment]
I think there might be a problem with this release. SBT is telling me that `specs2-scalacheck-binding_2.12.0-RC2` depends on `scalacheck_2.12.0-RC1`:

```
[error] Modules were resolved with conflicting cross-version suffixes in {file:/home/lars/proj/typelevel/discipline/}disciplineJVM:
[error]    org.scalacheck:scalacheck _2.12.0-RC2, _2.12.0-RC1
```

**Comment by adriaanm** [comment]
Due to https://github.com/etorreborre/specs2/blob/master/project/depends.scala#L34

**Comment by etorreborre** [comment]
That's because ScalaCheck was not available this morning. I am doing a `3.8.5.1` release right now to fix that.

**Comment by adriaanm** [comment]
Btw, is this a good reason to deviate from the standard versioning pattern? Perhaps just do `3.8.6`? I'm just saying, people are still joking about Scala 2.9.1-1 :-)

**Comment by etorreborre** [comment]
Just read your comment, the [cat is already out of the bag...](https://oss.sonatype.org/content/repositories/releases/org/specs2/specs2-scalacheck_2.12.0-RC2/3.8.5.1/specs2-scalacheck_2.12.0-RC2-3.8.5.1.pom). I would generally go `3.8.6` for minor bug fixes and improvements. In the case of `3.8.5` I think it can already be used just by forcing the right version of ScalaCheck using sbt. This is why I decided to go for `3.8.5.1`. 

That being said I have never been very good at naming versions so point taken.

**Comment by larsrh** [comment]
@etorreborre While you're at it, could you do a cross-release for `3.8.5.1` for 2.11 and 2.10 too?

**Comment by etorreborre** [comment]
@larsrh thanks for the reminder, done.

**Comment by larsrh** [comment]
Thanks!

**Comment by rossabaker** [comment]
I'm sad to even ask this: would it be possible to get a 3.8.5.1-scalaz-7.1?  I can probably wait for Scala 2.12.0-final, but it would be nice to have at least one version of Specs2 that works with Scalaz-7.1 on 2.12.

**Comment by etorreborre** [comment]
@rossabaker I just published `3.8.5.1-scalaz-7.1` cross Scala versions.
